src/routes/console/project-[project]/messaging/wizard/pushFormList.svelte

```diff
@@ -163,7 +163,11 @@
             </Modal>
         </div>
         <form class="form">
-            <Label>Custom data <span class="u-color-text-gray">(Optional)</span></Label>
+            <FormItem>
+                <Label
+                    tooltip="Deliver supplementary data alongside the notification, which remains hidden from the user's view but can be accessed by your application."
+                    >Custom data <span class="u-color-text-gray">(Optional)</span></Label>
```

Comment:
The `Label` component has an `optionalText` variable that we can use for the `(Optional)` text instead of using the span :)

Reply:
When I switched to using `optionalText` the spacing between the text seemed too big. That means we need to change the margin, right?

<img width="450" alt="image" src="https://github.com/appwrite/console/assets/1477010/a7aa6f56-cc4c-4ab5-be82-d0d0e86def4d">